apps/ckit-app/src/components/Wallet/WalletChange.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Col, Empty, Row } from 'antd';
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import { useSigner, WalletContainer } from 'containers/WalletContainer';
+
+const NervosExploreUrl = 'https://explorer.nervos.org/aggron/address/';
+
+export const WalletChange = observer(() => {
+  const { wallets, currentWalletIndex, setModalVisible } = WalletContainer.useContainer();
+  const wallet = currentWalletIndex === null ? undefined : wallets[currentWalletIndex];
+  const signer = wallet?.getSigner();
+  const { address } = useSigner(signer);
+  if (!address) return <Empty />;
+  const truncatedAddress = truncateMiddle(address, 6);
```

Comment:
```suggestion
  const truncatedAddress = truncateMiddle(address, 8);
```

6 may not enough since a ckb address always starts with `ckb` or `ckt`, maybe we should use 8

Reply:
fixed